engine/classes/Elgg/Database/EntityTable.php

```diff
@@ -1221,7 +1196,13 @@ public function getEntityAttributeWhereSql(array $options = []) {
 		}
 
 		// @todo the types should be defined somewhere (as constant on \ElggEntity?)
-		if (!in_array($type, ['group', 'object', 'site', 'user'])) {
+		if (!in_array($type, [
```

Comment:
This is actually PHPStorm formatting, I am not sure how to tell it to keep it as is. There were many changes in this file. I can rever some of these.